src/UI/Pages/SignIn/SignIn.tsx

```diff
@@ -46,36 +46,52 @@ const SignIn: React.FC = () => {
                         repudiandae inventore rem harum officiis. Aut!
                     </Paragraph>
                 </div>
-                <div className="inputsContainer">
-                    <div className="inputContainer">
-                        <InputLabel text="Email" />
-                        <Input
-                            placeholder="Email Address"
-                            value={email}
-                            onChange={({ target }) => loginInputHandler('email', target.value)}
-                        />
-                    </div>
-                    <div className="inputContainer">
-                        <InputLabel text="Password" />
-                        <Input
-                            placeholder="Password"
-                            type="password"
-                            value={password}
-                            onChange={({ target }) => loginInputHandler('password', target.value)}
-                        />
-                    </div>
-                </div>
-                <div className="forgetPasswordText">
-                    <strong>Forgot password</strong>
-                </div>
-                <div className="actionsContainer">
-                    <div className="buttonContainer">
-                        <ButtonP onClick={submit}>Sign in</ButtonP>
-                    </div>
-                    <div className="loginText">
-                        I don&apos;t have an account <strong onClick={() => navigate('../signup')}>Sign Up</strong>
-                    </div>
-                </div>
+                <Formik
+                    initialValues={signInForm}
+                    onSubmit={(values) => submit(values)}
```

Comment:
> onSubmit={(values) => submit(values)}

I think you can just write `onSubmit={submit}`
By doing so you can save a function declaration.